docs/QuickStart-GettingStarted.md

```diff
@@ -1,10 +1,10 @@
 ---
-id: getting-started
+id: quick-start-getting-started
 title: Getting Started
 layout: docs
 category: Quick Start
-permalink: docs/getting-started.html
-next: tutorial
+permalink: docs/quick-start-getting-started.html
```

Comment:
Can you avoid changing the link? This is going to mess up with google results and people's bookmarks. We should not change urls if possible and in this case you don't really need to, just make the header link to the other one.